transport/src/test/java/io/netty/channel/pool/SimpleChannelPoolTest.java

```
@@ -242,4 +244,124 @@ public void initChannel(LocalChannel ch) throws Exception {
         channel2.close().syncUninterruptibly();
         group.shutdownGracefully();
     }
+
+    @Test
+    public void testClose1() throws Exception {
+        EventLoopGroup group = new LocalEventLoopGroup();
+        LocalAddress address = new LocalAddress(LOCAL_ADDR_ID + ".SimpleChannelPoolTest.testClose1");
+        Bootstrap cb = new Bootstrap();
+        cb.remoteAddress(address);
+        cb.group(group).channel(LocalChannel.class);
+
+        ServerBootstrap sb = new ServerBootstrap();
+        sb.group(group).channel(LocalServerChannel.class).childHandler(
+            new ChannelInitializer<LocalChannel>() {
+                @Override
+                public void initChannel(LocalChannel ch) throws Exception {
+                    ch.pipeline().addLast(new ChannelInboundHandlerAdapter());
+                }
+            });
+
+        // Start server
+        Channel sc = sb.bind(address).sync().channel();
+        CountingChannelPoolHandler handler = new CountingChannelPoolHandler();
+
+        int count = 256;
+
+        SimpleChannelPool pool = new SimpleChannelPool(cb, handler);
+
+        ConcurrentSet<Channel> concurrentSet = new ConcurrentSet<Channel>();
+
+        for (int i = 0; i < count; ++ i) {
+            Future<Channel> future = pool.acquire();
+
+            try {
+                future.get();
+            } catch (Exception e) {
+                Assert.assertTrue(false);
```

why not let it just throw ?